var/spack/repos/builtin/packages/py-libensemble/package.py

```diff
@@ -72,12 +72,11 @@ def cache_test_sources(self):
     def run_tutorial_tests(self, exe):
         """Run example stand alone test"""
 
-        test_dir = join_path(
-            self.test_suite.current_test_cache_dir,
-            "examples",
-            "calling_scripts",
-            "regression_tests",
-        )
+        root_test_dir = self.install_test_root
+        if self.test_suite is not None:
+            root_test_dir = self.test_suite.current_test_cache_dir
+
+        test_dir = join_path(root_test_dir, "examples", "calling_scripts", "regression_tests")
```

Comment:
The user should not have to set `self.test_suite` as that is supposed to be provided by Spack core.

We would prefer they not be aware of or use `install_test_root`, especially for building or writing tests.  That is why we provided helper methods to copy cached test-related files.  Otherwise, running such tests result in permission-related failures on read-only file systems and facility deployments.  And I'm sure there are at least a few packages that have not been updated to stop using that property.